.github/workflows/bump-version.yaml

```diff
@@ -34,13 +34,16 @@ jobs:
           git_email: commitizen-github-action[bot]@users.noreply.github.com
           github_token: ${{ secrets.GITHUB_TOKEN }}
           commitizen_version: 2.24.0 # Keep in sync with pyproject.toml.
-      - name: Check out slack-templates.
-        uses: actions/checkout@v3.0.1
+      - name: Send Slack notification with job status (self-test).
+        if: ${{ github.repository == 'ScribeMD/slack-templates' }}
+        uses: ./
         with:
-          repository: ScribeMD/slack-templates
-          path: slack-templates
-      - name: Send Slack notification with job status.
-        uses: ./slack-templates/
+          bot-token: ${{ secrets.SLACK_BOT_TOKEN }}
+          channel-id: ${{ secrets.SLACK_ACTIONS_CHANNEL_ID }}
+          template: result
```

Comment:
See rhysd/actionlint#133. I don't feel that using an anchor and alias to eliminate the duplication warrants disabling the overall very helpful Actionlint, and I couldn't find a way to suppress an individual rule.